package.json

```diff
@@ -51,6 +51,7 @@
   "dependencies": {
     "camelcase": "4.0.0",
     "chalk": "1.1.3",
+    "didyoumean2": "1.3.0",
```

Comment:
Maybe this: https://github.com/aceakash/string-similarity
I will give it locally a go